glean-core/src/upload/mod.rs

```diff
@@ -133,8 +136,14 @@ impl PingUploadManager {
             .write()
             .expect("Can't write to pending pings queue.");
         match queue.pop_front() {
-            Some(request) => PingUploadTask::Upload(request),
-            None => PingUploadTask::Done,
+            Some(request) => {
+                log::info!("New upload task! {}", &request.uuid);
```

Comment:
```suggestion
                log::info!("New upload task with id {}", request.uuid);
```

Slight rephrasing to make clear that this is the request's id, will help in the future when starring at it.
(this should automatically use a reference anyway)